src/components/Form.jsx

```diff
@@ -0,0 +1,96 @@
+
+
+import { useFormik } from 'formik';
+import { basicSchema } from '../schemas';
+import './Form.css'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClipboardCheck } from '@fortawesome/free-solid-svg-icons';
+import Show from './Show';
+import Navbar from './Navbar';
+import { useState } from 'react';
+
+const FormElement =styled.form`
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+background: rgba(51, 49, 49, 0.356);
+padding: 25px;
+border-radius: 10px;
+color:white;
+width: fit-content;
+margin: 10% 30%;
+`
+const InputElement =styled.input`
+border-radius: 5px;
+width: 98%;
+height: 30px;
+margin: 20px 0px;
+border-width: 0;
+`
+const ButtonElement=styled.button`
+border-radius: 5px;
+width: 100%;
+height: 30px;
+background: rgba(183, 166 , 178 , 0.7);
+border-width: 0;
+color:white;
+opacity: 0.7;
+transition: opacity 0.3s ease; 
+&:hover {
+  opacity: 1; 
+}
+`
+function Form() {
+
+    const [isFormVisible, setIsFormVisible] = useState(true)
+
+
+    const onSubmit= (values)=>{
+        console.log(values)
+        setIsFormVisible(false);
+
+      
+    }
+
+
+
+    const {values,errors,handleChange,handleBlur,touched,handleSubmit}=useFormik({
+        
+        initialValues:{
+            name:"",
+            description:""
+        
+        },
+        validationSchema:basicSchema,
+        onSubmit,
+          
+    });
+
+
+  return (
+  <>
+    <Navbar/>
```

Comment:
if the navbar needs to be visible inside all the components, we can add it to the App.tsx file